filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -285,6 +285,10 @@ void OpenGLDriver::bindSampler(GLuint unit, GLuint sampler) noexcept {
     mContext.bindSampler(unit, sampler);
 }
 
+void OpenGLDriver::setPushConstant(backend::ShaderStage stage, uint8_t index,
+        backend::PushConstantVariant value) {
```

Comment:
The implementation here would be to call `glUniform{i|f}()` with the correct location from the index.
That location would have been pre-cached during init from the name. Easy.

Reply:
done